### PR TITLE
Added logger. Added screenshot. Added instructions. Added assertion for Python version.

### PR DESCRIPTION
**Pull request by ltfschoen**

### CODE REVIEW

File: 2_isolation/README.md
Comment (madhavajay) [inline 2_isolation/README.md]: This is handled by pipenv install in the main README.md instructions. Pipenv replaces both virtualenv and pip. Its awesome and the future: https://github.com/kennethreitz/pipenv

Also the python 3.6 requirement is already specified in the main README.md and the tests with Travis CI are set to python 3.6.

File: 2_isolation/README.md
Comment (madhavajay) [inline 2_isolation/README.md]: Awesome, a screenshot on the main README.md is good idea. Maybe change to the img folder so we dont have two image folders. Also I have made a screenshot with the same terminal theme on my machine and ill upload it over the top once iv merged your PR so the colours match.

File: 2_isolation/play.py
Comment (madhavajay) [inline 2_isolation/play.py]: I believe the code might well work in python 3.5, so maybe better to not restrict it in case someone can't run python 3.6.

File: 2_isolation/play.py
Comment (madhavajay) [inline 2_isolation/play.py]: Run the files through mypy and see what type warnings they suggest. They can't all be fixed because of the main Udacity code being untyped, but in the case of this new file, theres a few changes to make like
```python
def main(): -> None:
```

**Review by madhavajay — CHANGES_REQUESTED** [review]
Looks good, thanks for the help on cleaning it up and providing instructions to users. Ill merge after those few adjustments.

File: 2_isolation/play.py
Comment (ltfschoen) [reply 2_isolation/play.py]: I've changed it to 3.0.0 to allow any Python version >= 3.0

File: 2_isolation/README.md
Comment (ltfschoen) [reply 2_isolation/README.md]: I've moved the screenshot to the "main" README.md, and moved the screenshot directory and file into the existing img folder

File: 2_isolation/README.md
Comment (ltfschoen) [reply 2_isolation/README.md]: I agree with your suggestion, but I've included a link to the "main" README.md incase users overlook it and just directly go to 2_isolation/README.md (like I did).

FYI, I had trouble using pipenv myself due to conflicts, but I created an [Pipenv issue](https://github.com/kennethreitz/pipenv/issues/288) and working through trying to resolve it

**Comment by madhavajay** [comment]
Looks awesome! I'm particularly impressed with the stubs/timeit.pyi typeshed. I was meaning to look into how to do that so thanks for the example. 😎

Thanks for your help!

**Pull request merged**
